### ISSUE TITLE
color-contrast fails for monaco-editor when renderWhitespace set to true [WCAG1.4.3 test]

### ISSUE THREAD

**Issue by madhuv-msft** [microsoft/monaco-editor#1554]
**monaco-editor version:** Latest
**Browser:** Chrome latest
**OS:** Windows

**Steps or JS usage snippet reproducing the issue:**
1. go to https://microsoft.github.io/monaco-editor/playground.html
2. add `renderWhitespace: "all"` in the options
3. use value as `value: "function hello() {\n     alert('Hello world!');\n}",`
4. click on 'Run', now whitespaces are shown as mentioned below
![image](https://user-images.githubusercontent.com/26214977/63018980-cb437880-beb7-11e9-8170-19c3b67cb3d9.png)
5. Now install 'Accessibility Insights for Web' extension in chrome from https://chrome.google.com/webstore/detail/accessibility-insights-fo/pbjjkligggfmakdaogkfomddhfmpjeni and run 'fastpass', you can see the following error for the 
![image](https://user-images.githubusercontent.com/26214977/63019413-e5ca2180-beb8-11e9-8544-b4bdb883850b.png)

if i remove this option of 'renderWhitespace', then this color-contrast error is not coming.

**Comment by alexdima** [comment]
There is a special theme called the high contrast theme which we ship with and which can be used by embedders. Please see https://github.com/microsoft/monaco-editor/wiki/Accessibility-Guide-for-Integrators
